src/main/java/seedu/address/logic/commands/job/FindJob.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.logic.commands.job;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.function.Predicate;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.job.Job;
+
+/**
+ * Finds and lists all jobs in address book whose name contains any of the argument keywords or matches give id.
+ * Keyword matching is case insensitive.
+ */
+public class FindJob extends Command {
+
+    public static final String COMMAND_WORD = "findjob";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all job whose job title contains"
```

Comment:
Maybe we should specify that the command can only take in either a jobtitle keyword, or an id?